app/sidebars.js

```diff
@@ -110,6 +110,18 @@ module.exports = {
         }
       ]
     },
+    {
```

Comment:
I think it is best to put this doc about Peppol directly under and as last item in the menu category 'API documentation'.

For that we also need to update the [category page](https://standard.setu.nl/docs/api/). It now describes the HTTP methods, which is not needed because it is general knowledge. This category page can instead explain that for the SETU messages we have the option to implement the OAS specifications provided by SETU and for the Purchase to Pay messages the PEPPOL infrastructure is also an option.

There is one note regarding the `PUT` operation on this page. Let's move that to a seperate new page under the usage notes section. Very minimal, very short, only about this interpretation of the PUT operation.